AppWidget/glance-widget/src/main/java/com/example/android/glancewidget/TodoListGlanceWidget.kt

```diff
@@ -79,10 +83,12 @@ class TodoListGlanceWidget : GlanceAppWidget() {
         Column(
             modifier = GlanceModifier
                 .fillMaxSize()
-                .background(day = Color.White, night = Color.DarkGray)
+                .background(
+                    day = colorAttrResource(R.color.m3_sys_color_dynamic_light_background),
```

Comment:
If you resolve the color your would not get the dynamic part. Also, I believe this one is only available for v31.

While we don't have a better way to define themes, you would need to create a color resource and define each value for:

values/
values-night/
values-night-v31/
values-v31

Each of them using the mapped m3 color

Reply:
Thanks, done.